psi4/src/psi4/lib3index/dfhelper.cc

```diff
@@ -3379,7 +3383,7 @@ void DFHelper::compute_wK(std::vector<SharedMatrix> Cleft, std::vector<SharedMat
             first_transform_pQq(nocc, bcount, block_size, wMp, T2p, Crp, C_buffers);
 
             // compute wK
-            C_DGEMM('N', 'T', nbf_, nbf_, nocc * block_size, 1.0, T2p, nocc * block_size, T1p, nocc * block_size, 1.0,
+            C_DGEMM('N', 'T', nbf_, nbf_, nocc * block_size, 1.0, T1p, nocc * block_size, T2p, nocc * block_size, 1.0,
```

Comment:
This line right here is what I recommend for backporting and what fixes the bug.